tmk_core/protocol/lufa/lufa.c

```diff
@@ -559,27 +554,26 @@ static uint8_t keyboard_leds(void) {
  * FIXME: Needs doc
  */
 static void send_keyboard(report_keyboard_t *report) {
-    /* Select the Keyboard Report Endpoint */
-    uint8_t ep   = KEYBOARD_IN_EPNUM;
-    uint8_t size = KEYBOARD_REPORT_SIZE;
-
     /* If we're in Boot Protocol, don't send any report ID or other funky fields */
     if (!keyboard_protocol) {
-        send_report(ep, &report->mods, 8);
+        send_report(KEYBOARD_IN_EPNUM, &report->mods, 8);
     } else {
-#ifdef NKRO_ENABLE
-        if (keymap_config.nkro) {
-            ep   = SHARED_IN_EPNUM;
-            size = sizeof(struct nkro_report);
-        }
-#endif
-
-        send_report(ep, report, size);
+        send_report(KEYBOARD_IN_EPNUM, report, KEYBOARD_REPORT_SIZE);
     }
 
     keyboard_report_sent = *report;
 }
 
+/** \brief Send NKRO
+ *
+ * FIXME: Needs doc
```

Comment:
nit: while we're at it we could add a doc string here?

Reply:
I don't see much point TBH. My goal is to get rid of all these functions and just have the USB "backends" expose `send_report()`.

Reply:
That is also fine with me, imho wouldn't hurt to delete the lines in this case.